astroplan/core.py

```diff
@@ -37,6 +28,9 @@
 
 #__doctest_requires__ = {'*': ['scipy.integrate']}
 
+from astropy.utils import iers
+iers.IERS.iers_table = get_recent_IERS_A_table()
```

Comment:
right now this functions gets called *at import*.  What about instead having it get called the first time the user actually *does something* that requires the `iers` tables?  Or is that too hard to keep track of?